lib/Jsrt/Jsrt.cpp

```diff
@@ -2932,15 +2932,13 @@ CHAKRA_API JsIdle(_Out_opt_ unsigned int *nextIdleTick)
     });
 }
 
-CHAKRA_API JsSetPromiseContinuationCallback(_In_ JsPromiseContinuationCallback promiseContinuationCallback, _In_opt_ void *callbackState)
+CHAKRA_API JsSetPromiseContinuationCallback(_In_opt_ JsPromiseContinuationCallback promiseContinuationCallback, _In_opt_ void *callbackState)
 {
-    return ContextAPINoScriptWrapper_NoRecord([&](Js::ScriptContext * scriptContext) -> JsErrorCode {
-        PARAM_NOT_NULL(promiseContinuationCallback);
-
-        scriptContext->GetLibrary()->SetNativeHostPromiseContinuationFunction((Js::JavascriptLibrary::PromiseContinuationCallback)promiseContinuationCallback, callbackState);
+    return ContextAPINoScriptWrapper_NoRecord([&](Js::ScriptContext *scriptContext) -> JsErrorCode {
+        scriptContext->GetLibrary()->SetNativeHostPromiseContinuationFunction((Js::JavascriptLibrary::PromiseContinuationCallback) promiseContinuationCallback, callbackState);
         return JsNoError;
     },
-    /*allowInObjectBeforeCollectCallback*/true);
+    /*allowInObjectBeforeCollectCallback*/ true);
```

Comment:
It's not too distracting here, but it's better to minimize the changes you make when completing a bug fix.  These sorts of formatting changes create more opportunities for merge conflicts and can hide the true spirit of the change.  If we want to fix these, I would do a separate PR to take care of them or at least split them out of the bug fix commit and push two separate commits.